css/selectors/matchesfunction.json

```diff
@@ -0,0 +1,62 @@
+{
+  "css": {
+    "selectors": {
+      "matchesfunction": {
+        "__compat": {
+          "description": "<code>:matches()</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:matches()",
+          "support": {
+            "webview_android": {
+              "version_added": "66",
+              "notes": "Equivalent functionality also available via the <code>:-webkit-any()</code> pseudo-class."
```

Comment:
I would merge it with :any():
[{
 "version_added": "66"
},
{
  "version_added": "12",  (value to be double-checked on :any() page)
  "alternate_name": "-webkit-any()"
}]